src/Umbraco.Core/Properties/AssemblyInfo.cs

```diff
@@ -14,6 +14,7 @@
 [assembly: InternalsVisibleTo("Umbraco.Web")]
 [assembly: InternalsVisibleTo("Umbraco.Web.UI")]
 [assembly: InternalsVisibleTo("Umbraco.Examine")]
+[assembly: InternalsVisibleTo("Umbraco.ModelsBuilder.Embedded")]
```

Comment:
why is this needed?

Reply:
We use `PanicException` + `UrlExtensions.IsClientSideRequest` from Core

Reply:
ok